### DISCUSSION TITLE
Documentation for operator upgrade from 1.x to 24.x

### DISCUSSION THREAD

**Discussion by josedev-union** [apache/doris-operator#302] [General]
Hi folks.
I've been operating doris v2 clusters in production environments using doris-operator 1.6.
To upgrade these clusters to v3, I can see that need to upgrade doris-operator first. But I am afraid if new CRDs in v3 requires manual steps for ugprade.
Is there any documentation for upgrade for operator and clusters?
The main expectations are 
- operator upgrade should not affect existing clusters
- cluster version upgrade without downtime or with pre-deterministic duration

Thanks.

**Comment by liujie-yt** [comment]
In theory, there is no problem. Please update the CRD first
However, we have previously fixed an issue that caused a restart, so it is still recommended to test it in the testing cluster first
